web-client/src/presenter/actions/setDefaultTableSortAction.js

```diff
@@ -0,0 +1,31 @@
+import { ASCENDING, DESCENDING } from '../presenterConstants';
+import { state } from 'cerebral';
+
+/**
+ * sets the default sort based on the section picked.
+ *
+ * @param {object} providers the providers object
+ * @param {Function} providers.get the cerebral get function
+ * @param {object} providers.props the props passed to the sequence
+ * @param {object} providers.store the cerebral store object
+ */
+export const setDefaultTableSortAction = ({ get, props, store }) => {
+  // Story 9490 requires the sorting to only apply to the ADC Section boxes.  So these if statements exist to limit the scope for now.  These if statements can be removed in future stories that expand the scope.
```

Comment:
Y'all wrote some amazing tests that cover exactly what this comment is saying! Can this comment be removed in favor of using the tests as docs?

Reply:
Removed the comment!